### ISSUE TITLE
[IpadOS] Plz support external display full screen mode

### ISSUE THREAD

**Issue by BLOODTYP** [moonlight-stream/moonlight-ios#547]
Please use the airplay method like the full screen of the ott such as Netflix, or add a function that supports the full screen of the external display stage manager.

**Comment by cd419** [comment]
Yes I just tried this and hit the limitation of not being able to scale the app full screen when using stage manager and an external display. I’d like to take advantage of this to do some gaming on an external display which could be really cool.

**Comment by leobeosab** [comment]
It's been a while since I've done IOS dev but I'm running into this issue as well. I'll see if I can get stage manager + full screen working on an external display.

**Comment by leobeosab** [comment]
I was able to get fullscreen working on an external display by remove the requires full screen attribute from Limlight-Info.plist. It works fine for me with no drops in performance but I don't know if this would negatively affect other users.

![Screenshot 2023-02-23 at 2 37 56 PM](https://user-images.githubusercontent.com/1686221/221014473-9fde2281-964a-44d5-8750-827b9e8ed0e6.png)
Example of it working connected to a dock:
![03CA8D5D-1DE2-4F6C-A59F-E6E6BFE2CEBE](https://user-images.githubusercontent.com/1686221/221015207-bc7cc9d8-1ac7-408a-a98e-3f8c4ae61e65.jpg)

**Comment by BLOODTYP** [comment]
> It's been a while since I've done IOS dev but I'm running into this issue as well. I'll see if I can get stage manager + full screen working on an external display.

Thank you, but I don't know how to develop. How can I release this code as an official app?

**Comment by cd419** [comment]
@leobeosab Thanks for taking a look at this! Would love to be able to help test this but haven't delved into dev on iOS but I don't have a mac. Is there a way to build this directly on an iPad or windows / linux PC?

**Comment by patroza** [comment]
this may be the only thing keeping me from sunshine/moonlight, and leaves me stuck on steamlink

**Comment by ThuRoldao** [comment]
Any news on this matter? I would also love to be able to use the app in fullscreen on my external monitor

**Comment by jessedrakehovey** [comment]
Any news on this? Haven't seen a comment since 2024
